src/util/lengthCalculator.ts

```diff
@@ -0,0 +1,27 @@
+import * as L from 'leaflet';
+import IRoutePathLink from '../models/IRoutePathLink';
+
+const betweenCoordinates = (start: [number, number], end: [number, number]) => {
+    return L.latLng(start).distanceTo(L.latLng(end));
+};
+
+const fromPositions = (positions: [[number, number]]) => {
```

Comment:
could also use `positions: IRoutePathLink['positions']`

Reply:
True.. Somehow not liking the array of arrays implementation here and maybe would rather see an array of Latlng or ICoordinates, but that would make extra work for the code asking for this information.

Reply:
We could make issue of this, create some less messy logic for handling coordinates